scripts/scaffold-demo.sh

```
@@ -0,0 +1,505 @@
+<h1>React on Rails Demo</h1>
+<%= react_component("HelloWorld", props: { name: "World" }, prerender: false) %>
+EOF
+
+  # Add route
+  echo "  root 'hello_world#index'" >> "$DEMO_DIR/config/routes.rb"
+fi
+
+echo ""
+# Create comprehensive README
+echo -e "${YELLOW}📝 Creating README...${NC}"
+if [ "$DRY_RUN" = true ]; then
```

Shellcheck (0.11.0)

[error] Couldn't find 'fi' for this 'if'.

(SC1046)